resources/assets/templates/user/userItem.hbs

```diff
@@ -18,4 +18,10 @@
             <img src="/images/message.png"><div class="hidden-xs"><span>Message</span></div>
         </a>
     </div>
+
+    <div class="users-item-vote">
+        <a href='#/users/{{id}}/votes'>
+            <img src="/images/vote_img.png"><div class="hidden-xs"><span>My idea hubs</span></div>
```

Comment:
you no need links here, it should be separate section like My Topics, and should show list for currently logged in user